entrypoint.py

```diff
@@ -297,7 +350,7 @@ def main():
     UPLOAD = evaluate_boolean(os.getenv("UPLOAD", "True"))
     AUGMENT = evaluate_boolean(os.getenv("AUGMENT", "False"))
     ENRICH = evaluate_boolean(os.getenv("ENRICH", "False"))
-    OVERRIDE_SBOM_METADTA = evaluate_boolean(
+    OVERRIDE_SBOM_METADATA = evaluate_boolean(
```

Comment:
This should be updated on line 516 too as the type has been fixed here but the has old spelling there.